docs/user-guide/rules/list.md

```diff
@@ -141,6 +141,7 @@ Grouped first by the following categories and then by the [_thing_](http://apps.
 ### Custom property
 
 - [`custom-property-pattern`](../../../lib/rules/custom-property-pattern/README.md): Specify a pattern for custom properties.
+- [`declaration-block-no-duplicate-custom-properties`](../../../lib/rules/declaration-block-no-duplicate-custom-properties/README.md): Disallow duplicate custom properties within declaration blocks.
```

Comment:
Let's move this to the "Possible Errors -> Declaration Block" section (in alphabetical order).

I made a mistake in the blueprint, sorry.